src/components/RecentLinks.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { Fragment } from 'react';
+
+import Col from 'react-bootstrap/Col';
+import Row from 'react-bootstrap/Row';
+
+import { graphql } from 'gatsby';
+
+export default ({ links, showAmmount }) => {
+
+    const recentLinks = links.slice((-1 * showAmmount))
+
+    const shorten = (str, maxLen, separator = ' ') => {
+        if (str.length <= maxLen) return str;
+        return str.substr(0, str.lastIndexOf(separator, maxLen)) + '...';
+    }
+
+    return (
+        <div>
+            <div>
+                <p className="h2"> Recent Links</p>
+            </div>
+            {
+                recentLinks.map(link => (
+                    <Fragment>
+                        <a href={link.url} target="_blank" class="py-3 pl-8 pr-2 border-bottom d-block text-decoration-none">
```

Comment:
Hey there - I think you should be able to remove the `<Fragment>` surrounding this link, and I think you may need to add a `key={something}` prop to [keep react happy](https://reactjs.org/docs/lists-and-keys.html) here

Reply:
Hey! Sure. 
- `Fragment` removed.
- So the `key` is to give a unique name, and on native html it is not needed. But I went ahead and abstracted a RecentLinkItem, which is keyed.

Yes, it makes more sense this way now.

Let me know your thoughts!